### PR TITLE
Dnd 121 implementation

### PR DESCRIPTION
**Pull request by Maxenboy**
TitledCard :)

### CODE REVIEW

File: src/Molecules/TitledCard/TitledCard.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export const TitledCard: TitledCardComponent = props =>
+  props.title ? (
+    <Card>
+      <StyledTitle>
+        <Text.Title3>{props.title}</Text.Title3>
+      </StyledTitle>
+      {props.children}
+    </Card>
+  ) : (
+    <Card>{props.children}</Card>
```
Comment (jauniuse) [inline src/Molecules/TitledCard/TitledCard.tsx]: It feels for me that something is missing here.
As a consumer of this component I would want to use `TitledCard` in a same way as I would use `Card`, just with additional prop `title`, but this implementation restricts using `as` prop on `Card` for example. So such props need to be re-passed to `Card`.

We will have other components, like `CollabsibleCard` or `CardWithTabs`, but then they will need to have corresponding versions with titles also? Like `TitledCollabsibleCard`and `TitledCardWithTabs` - feels a bit too much. Maybe we should consider adding `title` prop to `Card` itself?

Comment (Maxenboy) [reply src/Molecules/TitledCard/TitledCard.tsx]: fixed

File: src/Molecules/TitledCard/TitledCard.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import R from 'ramda';
+import { Text, Card } from '../..';
+import { TitledCardComponent, Props } from './TitledCard.types';
+
+const StyledTitle = styled.div<Props>`
+  margin: ${({ theme }) => theme.spacing.unit(5)}px;
```
Comment (shmidt-i) [inline src/Molecules/TitledCard/TitledCard.tsx]: I think StyledTitle should only do margin-bottom, other spacing should be part of card

Comment (shmidt-i) [reply src/Molecules/TitledCard/TitledCard.tsx]: So maybe 
```
const StyledCard = styled(Card)`
  padding: ${({ theme }) => theme.spacing.unit(5)}px;
`

**Comment by shmidt-i** [comment]
And also, can we rename the component to `CardWithTitle` please?

File: src/Molecules/TitledCard/TitledCard.tsx (hunk 2 above)
Comment (Maxenboy) [reply src/Molecules/TitledCard/TitledCard.tsx]: fixed

**Pull request merged**